.github/workflows/linters.yml

```diff
@@ -0,0 +1,48 @@
+name: Linters
```

Comment:
## NOTE:
Since I cannot find the branch/PR with the first reviewers' comments, I would review this as the first review because there are no first review comments to check. Thanks

Reply:
please what do you expect me to change here